.tests/PSScriptAnalyzer/neo42PSScriptAnalyzerRules/neo42PSScriptAnalyzerRules.psm1

```diff
@@ -9,51 +9,51 @@ function Get-NxtPSUseCorrectTokenCapitalization {
 	.OUTPUTS
 	[Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.DiagnosticRecord[]]
 	#>
-	[CmdletBinding()]
 	[OutputType([Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.DiagnosticRecord[]])]
```

Comment:
To be consistent, the OutputType would also have to be adapted. List != [] 
Affects all customized functions.

Reply:
List can be cast into array. The defined output type is required by PSScriptAnalyzer to work. 
I precast the result into array now